webapp/channels/src/packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -298,7 +299,7 @@ export function createPost(post: Post, files: any[] = [], afterSubmit?: (respons
                 dispatch(batchActions(actions, 'BATCH_CREATE_POST_FAILED'));
                 return {error};
             }
-        }());
+        }()).then(() => {}));
```

Comment:
What's this change for?

Reply:
This isn't needed. It was my just my IDE complaining and I auto-fixed it. Reverted it now.